File: app/src/main/java/com/cv/anandmjoseph/anandcv/adapter/ViewPagerAdapter.java
package com.cv.anandmjoseph.anandcv.adapter;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

import com.cv.anandmjoseph.anandcv.ui.AboutmeOne;
import com.cv.anandmjoseph.anandcv.ui.Aboutmetwo;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Anand M Joseph on 12/29/2017.
 */

public class ViewPagerAdapter extends FragmentPagerAdapter {
    private final List<android.support.v4.app.Fragment> mFragmentList = new ArrayList<>();
    private final List<String> mFragmentTitleList = new ArrayList<>();

    public ViewPagerAdapter(FragmentManager manager) {
        super(manager);
    }

    @Override
    public android.support.v4.app.Fragment getItem(int position) {
        return mFragmentList.get(position);
    }

    @Override
    public int getCount() {
        return mFragmentList.size();
    }

    public void addFragment(Fragment fragment, String title) {
        mFragmentList.add(fragment);
        mFragmentTitleList.add(title);
    }

    @Override
    public CharSequence getPageTitle(int position) {
        return mFragmentTitleList.get(position);
    }
}

File: app/src/main/java/com/cv/anandmjoseph/anandcv/adapter/ViewPagerAdapter.kt
package com.cv.anandmjoseph.anandcv.adapter

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter

import com.cv.anandmjoseph.anandcv.ui.AboutmeOne
import com.cv.anandmjoseph.anandcv.ui.Aboutmetwo

import java.util.ArrayList

/**
 * Created by Anand M Joseph on 12/29/2017.
 */

class ViewPagerAdapter(manager: FragmentManager) : FragmentPagerAdapter(manager) {
    private val mFragmentList = ArrayList<android.support.v4.app.Fragment>()
    private val mFragmentTitleList = ArrayList<String>()

    override fun getItem(position: Int): android.support.v4.app.Fragment {
        return mFragmentList[position]
    }

    override fun getCount(): Int {
        return mFragmentList.size
    }

    fun addFragment(fragment: Fragment, title: String) {
        mFragmentList.add(fragment)
        mFragmentTitleList.add(title)
    }

    override fun getPageTitle(position: Int): CharSequence? {
        return mFragmentTitleList[position]
    }
}
